.golangci.yml

```diff
@@ -38,7 +38,7 @@ linters:
     # - whitespace
     # - wsl
     # - gocognit
-    - nolintlint
+    # - nolintlint
```

Comment:
This checks to see if there are unused nolint directives or if they are set correctly. We should leave it, it forces us to also put a reason to the nolint directive to inform the next person.

Reply:
I see. I thought it just disabled using `//nolint`